src/index.ts

```diff
@@ -1,31 +1,30 @@
 import { WebSocketServer } from "ws";
 
-import Tetris from "./core/tetris";
-import * as TetrisConsts from "./constants/tetris";
+import * as Tetris from "./core/tetris";
```

Comment:
I think we should only import what we need with:

`import { Command, Tetris } from "./core/tetris";`